scripts/mkcloudhost/cloudfunc

```diff
@@ -45,3 +45,10 @@ function routedcloudpublicnet()
     base2=$(($base2 % 256))
     echo -n 10.$base1.$(($base2+$n-1))
 }
+
+function ipv6prefix()
+{
+    local n=$1
+    local mkch_id=$(mkch_id)
+    printf FD3B:F495:9651:%X%02X "$mkch_id" "$n"
```

Comment:
Usually an IPv6 address is lower-case.

Reply:
good catch Nick, technically ipv6 are hexadecimal so it shouldn't be case sensitive but IETF says they MUST be represented as lowercase https://tools.ietf.org/html/rfc5952#section-4.3